src/Strategy/Erlang/Rebar3Tree.hs

```diff
@@ -68,13 +101,19 @@ buildGraph deps = unfold deps subDeps toDependency
                  , dependencyTags = M.empty
                  }
 
+lookupName :: Text -> M.Map Text Text -> Text
```

Comment:
[nit] This function feels unnecessary to me but I don't feel strongly about it. If you do decide to keep it, you should move it up closer to where its used and maybe put it in a `where` block.